app/views/search/single_facility.html.erb

```diff
@@ -1,62 +1,122 @@
 <% provide(:title, "Single Facility") %>
 <div id="content" class="interior">
   <a id="content" href="#content" class="sr-only">Content starts here.</a>
-  <%= render 'layouts/breadcrumbs' %>
-</div> 
```

Comment:
@rickleegit We generally have the breadcrumbs match the URL structure, and I don't believe we plan to have a URL structure that includes the facility name; URL structure would be something like `/vets.gov/facility-locator/facility` with some parameters in the URL after that, so the breadcrumb for each facility, matching our current style, would be `Facility Locator > Facility`, right?  In this case `Facility` is analogous  to `Institution` on the GI Bill.  

I think we would both agree that to deliver this prototype a week from today, this is not our biggest problem though, so let's move forward and either way we do it this week will be fine.